vllm_ascend/utils.py

```diff
@@ -30,4 +32,4 @@ def try_register_lib(lib_name: str, lib_info: str = ""):
             if lib_info:
                 logger.info(lib_info)
     except Exception:
-        pass
+        logger.warning_once(exception_info)
```

Comment:
I am not sure print this log is a good idea or not, especially consider mindie-turbo is a optional requirement for vllm-ascend